keel-core/src/test/kotlin/com/netflix/spinnaker/keel/api/ClusterDeployStrategyTests.kt

```diff
@@ -28,11 +26,13 @@ internal class ClusterDeployStrategyTests : JUnit5Minutests {
 
   fun tests() = rootContext<Fixture> {
     context("highlander") {
-      fixture { Fixture(Highlander) }
+      fixture { Fixture(Highlander()) }
 
       test("serializes to JSON") {
-        expectThat(mapper.writeValueAsString(strategy))
-          .isEqualTo("""{"strategy":"highlander"}""")
+        expectThat<ObjectNode>(mapper.valueToTree(strategy)) {
+          path("strategy").textValue() isEqualTo "highlander"
+          path("considerOnlyAmazonHealth").booleanValue().isFalse()
```

Comment:
```suggestion
          path("considerOnlyInstanceHealth").booleanValue().isFalse()
```

Reply:
Hmm, why is this not failing?

Reply:
Oh, nasty. `JsonNode.booleanValue()` returns `false` by default, so if it's actually a `MissingNode` rather than a `BooleanNode` this test passes.